src/plugins/intel_cpu/src/utils/precision_support.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "precision_support.h"
+
+#include "ie_precision.hpp"
+#include "cpu/x64/cpu_isa_traits.hpp"
+#include "openvino/core/visibility.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+bool isSupported(const InferenceEngine::Precision& precision) {
```

Comment:
1. Does it make sence to rename into `isSupportedInferencePrecision`? My understanding is that it covers only inference precision use-cases.
2. If 1st item is applicable function should return false by default.
3. Can we reuse this utility in Config::readProperties?